lib/gui/app.js

```diff
@@ -94,13 +94,14 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   const currentVersion = packageJSON.version;
   const currentReleaseType = release.getReleaseType(currentVersion);
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates({
-    ignoreSleepUpdateCheck: currentReleaseType !== release.RELEASE_TYPE.PRODUCTION
+    releaseType: currentReleaseType,
+    lastSleptUpdateNotifier: settings.get('lastSleptUpdateNotifier'),
+    lastSleptUpdateNotifierVersion: settings.get('lastSleptUpdateNotifierVersion') || packageJSON.version
   });
 
   if (_.some([
     !shouldCheckForUpdates,
-    process.env.ETCHER_DISABLE_UPDATES,
-    currentReleaseType === release.RELEASE_TYPE.UNKNOWN
+    process.env.ETCHER_DISABLE_UPDATES
```

Comment:
Is this change "correct"? Previously if we had a `release.RELEASE_TYPE.UNKNOWN` then it would always skip the update check, but now it doesn't do that? If we have a `release.RELEASE_TYPE.UNKNOWN` then we can't know what to update to, can we?